websocket/connection/table.go

```diff
@@ -0,0 +1,56 @@
+package connection
+
+import (
+	"sync"
+)
+
+type Table struct {
+	m       sync.Mutex
+	connMap map[Identifer]Identifer
+	counter map[string]int
+	maxUser int
+}
+
+func NewTable(maxUser int) *Table {
+	return &Table{
+		m:       sync.Mutex{},
+		connMap: make(map[Identifer]Identifer),
```

Comment:
Yes, it's used as a set. Will change the value to `struct{}`. Combine it into single map is a good idea. But, in this case, it is a little more complicated because map in GO has zero value of nil. Which we will need to do nil check here and there. IMO the current approach is preferable.